Gemfile.lock

```diff
@@ -120,6 +120,3 @@ DEPENDENCIES
   rspec-puppet
   sshkey (= 1.9.0)
   webmock (~> 1.20.0)
-
-RUBY VERSION
```

Comment:
Why the modification of the Gemfile.lock?

Reply:
Snap :)

Was installing a dev VM in the background and it turns out that the script for that make changes to the working copy. Nasty. Should be gone now.